### PR TITLE
Adding changeset for document field changes

### LINKED ISSUE #504: Underline option throws Astro error
When using the option to underline text inside the document editor, Keystatic produces this markup inside the mdoc file: `{% u %}lorem ipsum{% /u %}`

I use the `format: { contentField: 'content' }` option.

Trying to render this via Astro's content collection throws an error:

```
 error   **src/content/kontaktCTA/index.mdoc** contains invalid content:
  - Undefined tag: 'u'
```
I am pretty sure there is no underline feature in markdoc, so disabling this by default  in the editor might be a good idea.

### PR DESCRIPTION
**Pull request by JedWatson**
These changes were merged to `main` without a changeset, so catching up with this PR (and will review branch protection rules tomorrow!)

**TL;DR** I've updated the document field defaults, and the associated documentation, as mentioned in #531

This closes #504

I'd like to spend some more time with the docs to better explain how to set up custom tags in Markdoc properly if you opt into the (now-off) additional formatting options, so #531 remains open until then, but this means we have one less footman in the meantime.

### CODE REVIEW

**Review by emmatown — APPROVED** [review]
It'll be kinda breaking in probably subtly annoying ways but I broadly agree
